.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,192 @@
+        current_chart_version=$(yq '.version' deployment/network-operator/Chart.yaml)
+        echo APP_VERSION=""$app_version""                    >> $GITHUB_ENV
+        echo VERSION=""$current_chart_version-$app_version"" >> $GITHUB_ENV
+    - if: github.ref_type == 'tag'
+      name: Determine versions (when git tag)
+      run: |
+        git_tag=${{ github.ref_name }}
+        app_version=$git_tag
+        chart_version=${git_tag:1}  # without the 'v' prefix
+        echo APP_VERSION=""$app_version"" >> $GITHUB_ENV
+        echo VERSION=""$chart_version""   >> $GITHUB_ENV
+    - if: github.ref_name == env.DEFAULT_BRANCH || github.ref_type == 'tag'
+      name: NGC setup and authentication
+      run: |
+        wget \
+          --no-verbose \
+          --content-disposition \
+          -O ngccli_linux.zip \
+          https://api.ngc.nvidia.com/v2/resources/nvidia/ngc-apps/ngc_cli/versions/3.41.4/files/ngccli_linux.zip
+        unzip -q ngccli_linux.zip
+        echo "./ngc-cli" >> $GITHUB_PATH
+
+        ngc-cli/ngc config set <<EOF
+          ${{ secrets.NVCR_TOKEN }}
+          json
+          nvstaging
+          mellanox
+          no-ace
+        EOF
+    - if: github.ref_name == env.DEFAULT_BRANCH || github.ref_type == 'tag'
+      name: Make package and push
+      env:
+        NGC_REPO: nvstaging/mellanox/network-operator
+      run: |
+        make chart-build chart-push
+    - if: github.ref_name == env.DEFAULT_BRANCH || github.ref_type == 'tag'
+      name: NGC logout
+      run: |
+        ngc config clear-cache
+        ngc config clear
+
+  ocp-bundle:
+    if: github.ref_type == 'tag'
+    needs:
+    - docker-build-push
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
+      DOWNSTREAM_REPO_OWNER: nvidia-ci-cd
+      UPSTREAM_REPO_OWNER: redhat-openshift-ecosystem
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}  # token must be explicitly set here for push to work in following step
+    - uses: docker/login-action@v3
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ secrets.NVCR_USERNAME }}
+        password: ${{ secrets.NVCR_TOKEN }}
+    - name: Determine version and tag
+      run: |
+        git_tag=${{ github.ref_name }}
+        echo VERSION_WITH_PREFIX=$git_tag        >> $GITHUB_ENV
+        echo VERSION_WITHOUT_PREFIX=${git_tag:1} >> $GITHUB_ENV  # without the 'v' prefix
+    - name: Lookup image digest
+      run: |
+        network_operator_digest=$(skopeo inspect docker://$REGISTRY/$IMAGE_NAME:$VERSION_WITH_PREFIX | jq -r .Digest)
+        echo $network_operator_digest | wc -w | grep 1  # verifies value not empty
+        echo NETWORK_OPERATOR_DIGEST=$network_operator_digest >> $GITHUB_ENV
+    - name: Make bundle
+      env:
+        DEFAULT_CHANNEL: ${{ env.VERSION_WITH_PREFIX }}
+        CHANNELS: stable,${{ env.VERSION_WITH_PREFIX }}
```

Comment:
The channels should `stable` and the version with vMAJOR.MINOR only.
e.g. v24.7 and not v24.7.0

Reply:
That makes sense, however it is not how it appears to have been managed historically ([for example in 24.4.1](https://github.com/maze88/certified-operators/blob/main/operators/nvidia-network-operator/24.4.1/metadata/annotations.yaml#L7-L8)).
Perhaps @e0ne can settle this?

Reply:
channels should be managed as @rollandf mentioned above.
that way patch version of the same vMAJOR.MINOR end up in the same channel.

Reply:
Thanks for confirming, I've corrected this.